api/v1beta1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the frrk8s v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=frrk8s.metallb.io
+package v1beta1
```

Comment:
what's the reason we're starting from beta?

Reply:
Because alpha means "we can change and break this" while I would try to avoid it